lib/generators/open_stax/rescue_from/install/install_generator.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails/generators'
+
+module OpenStax
+  module RescueFrom
+    class InstallGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def copy_templates
+        copy_file "any.html.erb", "app/views/errors/any.html.erb"
+      end
+
+      def copy_initializer
+        copy_file "rescue_from.rb", "config/initializers/rescue_from.rb"
+      end
+
+      def configure_environments
+        test_file = "test.rb"
+        test_middleware = <<-TEST
+  config.middleware.use ExceptionNotification::Rack, email: {
```

Comment:
I'll definitely look into a more elegant solution. I'm always up for that! I did the quickest thing that seemed okay to do to finish the generators, because I assumed it wouldn't get a lot of priority as a part of the assignment ; )